test/0098_validate_binary_search_tree_test.cc

```diff
@@ -1,20 +1,22 @@
 #include "0098_validate_binary_search_tree.h"
 
+using std::vector;
+
 TEST(S, is_valid_bst) {
     Solution s;
 
     EXPECT_EQ(s.isValidBST(
-        base::TreeNodeFactory<int>(
+        leetlib::TreeNodeFactory<int>(
```

Comment:
Consider add `using leetlib::TreeNodeFactory` to the beginning of this source file.